src/actions/drawingPointActions.ts

```diff
@@ -33,3 +33,6 @@ export const drawingMeasure = createAction('DRAWING_MEASURE')<{
   elevation?: boolean;
   position?: { lat: number; lon: number };
 }>();
+
+export const setSelectedIcon = createAction('SET_SELECTED_ICON')<string>();
```

Comment:
toto patrí do `src/actions/objectsActions.ts` keďže sa to teraz týka iba funkcie "objekty"